Cargo.lock

```diff
@@ -17,6 +17,15 @@ dependencies = [
  "memchr",
```

Comment:
Should `Cargo.lock` be tracked by git?
We gitgnore it in `scylla-rust-driver`, but maybe it's good too make sure that everyone that uses these benchmarks uses the same version of libraries.

Reply:
As a rule of thumb, projects which have binary artifacts do track `Cargo.lock`, and rust libraries do not: https://doc.rust-lang.org/cargo/faq.html#why-do-binaries-have-cargolock-in-version-control-but-not-libraries

> maybe it's good too make sure that everyone that uses these benchmarks uses the same version of libraries.

I agree, so let's keep it. We will naturally update it from time to time so that new releases of cql-stress should receive updates for the dependencies.